.pre-commit-config.yaml

```diff
@@ -17,7 +17,7 @@ repos:
       - id: isort
 
   - repo: https://gitlab.com/pycqa/flake8
-    rev: 4.0.1
+    rev: 3.9.2
```

Comment:
Isn't this a downgrade?

Reply:
I did with `pre-commit autoupdate`. it seems we were pointing to the wrong repo.